server/controllers/admin/locations/deleteLocations.js

```diff
@@ -1 +1,23 @@
+const {
+  deleteSingleRecord
+} = require("../../../database/queries/rentalRecord");
+
+module.exports = async function deleteLocations(req, res) {
+  const record = req.body;
+
+  try {
+    const { deletedCount } = await deleteSingleRecord(record.rentalId);
+
+    if (deletedCount !== 1) {
+      res.status(404).send("User not found");
```

Comment:
return `res` here so that it definitely won't hit the next one. That might be what was causing the "set headers twice" error, although not sure why changing to `.send` would get rid of it...